### DISCUSSION TITLE
notification after command execution via SMS?

### DISCUSSION THREAD

**Discussion by HladRom** [community/community#110657] [Accessibility]
### Select Topic Area

General

### Body

what does everyone think about this

**Comment by queenofcorgis** [comment]
Hi @HladRom , we're happy you're here! You are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone give you a relevant response.

**Comment by ghostinhershell** [comment]
Hi @HladRom, 

Thanks so much for reaching out in the community. Since there hasn't been any updates in this particular post, I'm going to go ahead and close it. If you have any follow-up questions or feedback, please feel free to leave your comment here and reopen this discussion. Thanks!
